### PR TITLE
seminars: add new schema type

### PR DESCRIPTION
**Pull request by michamos**
* INSPIR-3489
* Also updates `normalize_yaml.py` script to work on Python 3 instead of
  Python 2.

### CODE REVIEW

File: inspire_schemas/records/seminars.yml [hunk 1]
```diff
@@ -0,0 +1,263 @@
+$schema: http://json-schema.org/schema#
+additionalProperties: false
+properties:
+    $schema:
+        format: uri
+        minLength: 1
+        type: string
+    _bucket:
```
Comment (monaawi) [inline inspire_schemas/records/seminars.yml:8]: is this still needed?

File: inspire_schemas/records/seminars.yml [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        minItems: 1
+        type: array
+        uniqueItems: true
+required:
+- _collections
+- end_datetime
+- start_datetime
+- timezone
+- speakers
+- title
```
Comment (monaawi) [inline inspire_schemas/records/seminars.yml:261]: in the mockup, ```inspire_categories``` is also mandatory in the submission form. Should we make it required here as well?

File: inspire_schemas/records/seminars.yml [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+        items:
+            enum:
+            - Seminars
+            minLength: 1
+            type: string
+        minItems: 1
+        title: Collections to which this record belongs
+        type: array
+        uniqueItems: true
+    _private_notes:
```
Comment (monaawi) [inline inspire_schemas/records/seminars.yml:29]: are these the ```additional information``` in the form? Or does ```additional information``` correspond to ```public_notes```? If yes, then how do these get populated?

File: inspire_schemas/records/seminars.yml [hunk 4]
```diff
@@ -0,0 +1,263 @@
+$schema: http://json-schema.org/schema#
+additionalProperties: false
+properties:
```
Comment (monaawi) [inline inspire_schemas/records/seminars.yml:3]: In the form we also have the field ```institution```, I also saw there was a discussion about it on zulip. But I see you didn't add the ```institution``` property, do we not want it?

File: tests/unit/test_api.py [hunk 5]
```diff
@@ -167,3 +167,28 @@ def test_validate_raises_on_invalid_orcid():
...
+        '$schema': 'http://json-schema.org/schema#',
+        'type': 'string',
+        'format': 'timezone',
+    }
+
+    utils.validate(data, schema)
+
+
+def test_validate_raises_on_invalid_timezone():
+    data = "SevenKingdoms/KingsLanding"
```
Comment (monaawi) [inline tests/unit/test_api.py:185]: :laughing:

File: inspire_schemas/records/seminars.yml (hunk 3 above)
Comment (michamos) [reply inspire_schemas/records/seminars.yml:29]: additional information maps to public notes, private notes are private stuff only visible to curators (probably not needed here).

File: inspire_schemas/records/seminars.yml (hunk 2 above)
Comment (michamos) [reply inspire_schemas/records/seminars.yml:261]: :+1: makes sense

File: inspire_schemas/records/seminars.yml (hunk 1 above)
Comment (michamos) [reply inspire_schemas/records/seminars.yml:8]: :shrug: it's still there for all other record types, so I added it too.

File: inspire_schemas/records/seminars.yml (hunk 4 above)
Comment (michamos) [reply inspire_schemas/records/seminars.yml:3]: we are dropping `institution` in the end: https://inspirehep.zulipchat.com/#narrow/stream/122202-devel/topic/seminars/near/195704729

File: inspire_schemas/records/seminars.yml (hunk 1 above)
Comment (monaawi) [reply inspire_schemas/records/seminars.yml:8]: we don't have ```invenio_records_files``` anymore, I think we can safely remove it from here. I don't know why we have them in other record types.

Comment (monaawi) [reply inspire_schemas/records/seminars.yml:8]: I would still remove this as we definitely don't need it

File: inspire_schemas/records/seminars.yml (hunk 3 above)
Comment (monaawi) [reply inspire_schemas/records/seminars.yml:29]: We should either remove these if we don't need them or add a way of adding/updating them if we think they are useful

Comment (michamos) [reply inspire_schemas/records/seminars.yml:29]: they're there in all collections, even jobs, so I think it's better to keep it even though there's no easy way to edit them currently

File: inspire_schemas/records/seminars.yml (hunk 1 above)
Comment (michamos) [reply inspire_schemas/records/seminars.yml]: done

File: inspire_schemas/records/seminars.yml (hunk 3 above)
Comment (monaawi) [reply inspire_schemas/records/seminars.yml:18]: ok, you're right. Let's keep them then.

**Pull request merged**